tests/unit/model.py

```diff
@@ -97,7 +97,8 @@ def sample_obj(self):
 class TestMetadata(ModelTest):
 
     def sample_obj(self):
-        node = Node(label='node-99',
+        from haas.ext.obm.ipmi import Ipmi
+        return Node(label='node-99',
```

Comment:
I think you meant `node = Node(...` instead of `return Node(...`.

Which brings up the question - shouldn't this test be failing?